BUILDING.md

```diff
@@ -295,7 +295,7 @@ If you want to run the linter without running tests, use
 `make lint`/`vcbuild lint`. It will run both JavaScript linting and
 C++ linting.
 
-If you are updating tests and just want to run a single test to check it:
+If you are updating tests and just want to run a single test(e.g. `parallel/test-stream2-transform`) to check it:
```

Comment:
```suggestion
If you are updating tests and just want to run a single test (e.g. `test/parallel/test-stream2-transform.js`) to check it:
```

Probably also needs to be broken into two lines to satisfy the line length linter rule.